### PR TITLE
Fixtableheader 325

### PR DESCRIPTION
**Pull request by vijayakarumudi**
Closes AgileVentures/MetPlus_tracker#325

These files are updated through  #314 agency_people/_job_seekers
 agency_people/home.

### CODE REVIEW

**Comment by vijayakarumudi** [comment]
PR for review

File: app/views/jobs/_list_all.html.haml [hunk 1]
```diff
@@ -4,11 +4,11 @@
...
-            %td.strong Title
+            %th.strong Title
           - when :updated_at
-            %td.strong Last update
+            %th.strong Last update
           - when :poster
-            %td.strong Poster
+            %th.strong Poster
           - when :num_applicants
             %td.strong Applications
```
Comment (patmbolger) [inline app/views/jobs/_list_all.html.haml]: Looks like this line (13) was missed.

**Comment by patmbolger** [comment]
Reviewed PR - looks good except for one missing conversion, see comment.

**Comment by joaopapereira** [comment]
Looks good, please review Patrick's comments

**Comment by vijayakarumudi** [comment]
I updated my PR,

**Comment by joaopapereira** [comment]
Looks good going to merge

**Pull request merged**